<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPlainTextEdit" name="recvEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>串口选择</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="5">
            <widget class="QComboBox" name="protsCmb"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>波特率</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="4">
            <widget class="QComboBox" name="baudRateCmb"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>停止位</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="4">
            <widget class="QComboBox" name="stopBitsCmb"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>数据位</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="4">
            <widget class="QComboBox" name="dataBitsCmb"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>校验位</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="4">
            <widget class="QComboBox" name="parityCmb"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>串口操作</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="4">
            <widget class="QPushButton" name="btnOpenPort">
             <property name="text">
              <string>打开</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="3">
            <widget class="QPushButton" name="btnSaveData">
             <property name="text">
              <string>保存数据</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3" colspan="2">
            <widget class="QPushButton" name="btnClearData">
             <property name="text">
              <string>清除接受</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QCheckBox" name="hexDisplayChx">
             <property name="text">
              <string>16进制显示</string>
             </property>
            </widget>
           </item>
           <item row="8" column="2" colspan="3">
            <widget class="QCheckBox" name="checkDTR">
             <property name="text">
              <string>DTR</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QCheckBox" name="checkRTS">
             <property name="text">
              <string>RTS</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2" colspan="3">
            <widget class="QCheckBox" name="checkAutoSave">
             <property name="text">
              <string>自动保存</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0" colspan="2">
            <widget class="QCheckBox" name="checkTimeStamp">
             <property name="text">
              <string>时间戳</string>
             </property>
            </widget>
           </item>
           <item row="10" column="2" colspan="2">
            <widget class="QLineEdit" name="editTimeStamp">
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
           <item row="10" column="4">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>ms</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>单条发送</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QPlainTextEdit" name="editCode"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>多条发送</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QPlainTextEdit" name="plainTextEdit_3"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>传输协议</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QPlainTextEdit" name="plainTextEdit_4"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>帮助</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="btnSend">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearSend">
            <property name="text">
             <string>清除发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkTimer">
          <property name="text">
           <string>定时发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>周期：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timerPeriodEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editFilePath"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnOpenFile">
          <property name="text">
           <string>打开文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSendFile">
          <property name="text">
           <string>发送文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStopSend">
          <property name="text">
           <string>停止发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkSendHex">
          <property name="text">
           <string>16进制发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checNewLine">
          <property name="text">
           <string>发送新行</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>258</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
